README.md

```diff
@@ -43,4 +43,5 @@ Change these options in the workflow `.yml` file to meet your GitHub project nee
 | `api-token`                       | [Account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) | `${{ secrets.CODACY_API_TOKEN }}`     |
 | `project-token`                   | [Project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) | `${{ secrets.CODACY_PROJECT_TOKEN }}` |
 | `coverage-reports`                | Optional comma-separated list of reports to send                                       | `''`                                  |
+| `language`                        | Optionally force your project language                                                 | `''`                                  |
```

Comment:
In fact, this ends up being the programming language that the coverage report is associated with, correct? Since a repository can have multiple languages and we can upload multiple coverage reports [specifying different languages for each one](https://docs.codacy.com/coverage-reporter/#multiple-languages).

I suggest:

```suggestion
| `language`                        | Optionally force associating a language with your coverage report                      | `''`                                  |
```